lib/app_web/views/item_view.ex

```diff
@@ -0,0 +1,56 @@
+defmodule AppWeb.ItemView do
+  use AppWeb, :view
+
+  # add class "completed" to a list item if item.status=1
+  def complete(item) do
+    case item.status do
+      1 -> "completed"
+      # empty string means empty class so no style applied
+      _ -> ""
+    end
+  end
+
+  #  add "checked" to input if item.status=1
+  def checked(item) do
+    case item.status do
+      1 -> "checked"
+      # empty string means empty class so no style applied
+      _ -> ""
+    end
+  end
+
+  # returns integer value of items where item.status == 0 (not "done")
+  def remaining_items(items) do
+    Enum.filter(items, fn i -> i.status < 1 end) |> Enum.count()
```

Comment:
I don't think we need the `Enum.filter` here as we can use the anonymous function directly in count:
```suggestion
    Enum.count(items, fn i -> i.status < 1 end)
```

Reply:
@SimonLab this is a _good_ improvement/simplification provided we explain that `Enum.count/2` can take a "filter" function as it's second param. Otherwise I would leave it as is and let a member of the community suggest the refactor. 👍